src/Microsoft.TemplateEngine.Edge/Settings/MountPointManager.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace Microsoft.TemplateEngine.Edge.Settings
 {
+    [System.Obsolete]
```

Comment:
Please comment what alternative should be used.

Reply:
This was now deleted since it was internal already, but interface has obsolete message.